src/styles/blocks/stars--5.css

```diff
@@ -0,0 +1,44 @@
+.stars--5 {
+  display: flex;
+  margin-top: 4px;
+}
+
+.stars--5 .stars__star:nth-child(1) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
```

Comment:
Consider using a class for common styles to avoid repetition. Each `.stars__star` has the same `display`, `background-image`, `width`, and `height` properties. You can extract these to a single class to make your CSS more maintainable.